scripts/mesh-point/mesh-point

```diff
@@ -6,30 +6,35 @@ function isMeshable {
     # Check to see if driver reports missing mesh point support
     if  [ -z "$(iw phy phy$1 info | grep 'mesh point')" ]; then
         result='' # Indicate interface is not meshable
-    # XRADIO driver reports Mesh Point support incorrectly (there is no support for it)
-    elif [ "$(basename $(readlink /sys/class/net/$interface/device/driver))" == 'xradio_wlan' ]; then	    
+    # XRADIO driver reports Mesh Point but does not actually work
+    elif [ "$(basename $(readlink /sys/class/net/$interface/device/driver))" == 'xradio_wlan' ]; then
         result='' # Indicate interface is actually not meshable
     else
-        result='1' # Indicate interface is  meshable
+        result='1' # Indicate interface is meshable
     fi
 
     echo $result
 }
 
-
 set -e
 
 # Set wireless regulatory domain
 sudo iw reg set CA
 
-# Select physical device that supports 802.11s Mesh Point
-if ! [ -z $(isMeshable 0) ]; then
-    mesh_phy=phy0
-    mesh_dev=$(iw dev | grep phy#0 -A 1 | grep Interface | awk '{print $2}')
-elif ! [ -z $(isMeshable 1) ]; then
-    mesh_phy=phy1
-    mesh_dev=$(iw dev | grep phy#1 -A 1 | grep Interface | awk '{print $2}')
-else
+# Find first meshpoint device you can
```

Comment:
It's better to use `802.11s Mesh Point` to be clear. The word meshpoint gets used in so many different context.